packages/blade/src/components/Popover/PopoverContext.ts

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { throwBladeError } from '~utils/logger';
+
+type PopoverContext = {
+  close: () => void;
+  defaultInitialFocusRef: React.RefObject<HTMLElement>;
+  titleId: string;
+} | null;
+const PopoverContext = React.createContext<PopoverContext>(null);
+
+const usePopoverContext = (): NonNullable<PopoverContext> => {
+  const context = React.useContext(PopoverContext);
+
+  if (__DEV__) {
+    if (!context) {
+      throwBladeError({
+        message: `PopoverInteractiveWrapper must be used within Popover`,
```

Comment:
is this hook for popover context or popoverinteractive wrapper? the error and the context hook name are different which could be misleading

Reply:
Fixed.